docs/release-notes/2.31.0.md

```diff
@@ -0,0 +1,154 @@
+
+## Release Highlights
+
+
+#### Grafana 9.1 in PMM 2.31.0
+
+We have upgraded Grafana to version 9.1 in PMM 2.31.0. With this upgrade, you can get a simplified user interface, enhanced visualizations, and a default Alerting experience. For more information, see What’s new in Grafana.
+ 
+The following features have been introduced with the upgrade to Grafana to 9.1:
+ 
+- **Expandable navbar (main menu)**
+ 
+    For an enhanced user experience and better accessibility, the main menu has been expanded with crucial elements such as starred dashboards, system, MongoDB, MySQL, PostgreSQL, ProxySQL, and HAProxy.
+ 
+- **Prometheus Query Builder**
+
+    A new UI query builder has been introduced, by which you can write Prometheus queries effortlessly. Prometheus queries can be complex to grasp, but with this Grafana release in PMM, it becomes easier. 
+
+- **Improvements to dashboard search**
+    
+    The dashboards can now be searched by panel title as well as dashboard title. Also, the overall performance has been enhanced as querying and sorting have become much faster.
+
+- **Default unified alerting**
+
+    Grafana Alerting is now on by default.
+
+## Vacuum monitoring dashboard
+
+!!! caution alert alert-warning "Important"
+    These experimental dashboards are subject to change. It is recommended to use these dashboards for testing purposes only.
+
+
+A new experimental dashboard has been introduced to overcome the difficulties related to vacuum by providing them with the right and timely insights.
+
+## Podman GA
+
+We are excited to announce the General Availability (GA) of Podman support for Deploying PMM 2.31.0. We had introduced it in 2.29.0 as a preview feature, but now we are production ready with this feature.
+
+
+## New Alerting
+
+Percona Alerting is a solution based on Grafana Alerting with the addition of simplicity of Templates usage.  
+This will enable users to use Alerting without deep knowledge of metrics details and consume the best knowledge from Percona Experts who created Alerting Templates. 
+
+
+
+### Database as a Service (DBaaS)
+
+In our constant endeavor and focus on an enhanced user experience, in PMM 2.31.0, we have simplified the deployment and configuration of DBaaS as follows:
+
+- With PMM 2.31.0, you can easily add a DB cluster from the newly created K8s cluster. All the DB cluster window fields are auto-populated with the values based on the existing K8s cluster. 
+
+- For PMM 2.31.0, while accessing DbaaS, if you have an existing Kubernetes cluster configured for DBaaS, you will be automatically redirected to the DB Cluster page. Otherwise, you would be redirected to the Kubernetes Cluster page.
+
+- We have added support for the latest (?) released Percona Operator for MongoDB (link to operator releases).
```

Comment:
Probably need to be removed because https://jira.percona.com/browse/PMM-9989 might not be released.  

@gen1us2k, please let @rasika-chivate know the final result.

Reply:
@gen1us2k Please reaffirm...

Reply:
The issue is still in progress. We do not support the 1.12 PSMDB operator yet.

Reply:
Will remove it from the RN..

Reply:
Done